src/ChunkMap.cpp

```diff
@@ -2100,17 +2100,17 @@ void cChunkMap::Tick(std::chrono::milliseconds a_Dt)
 
 
 
-void cChunkMap::TickBlock(int a_BlockX, int a_BlockY, int a_BlockZ)
+void cChunkMap::TickBlock(const Vector3i a_BlockPos)
 {
+	auto chunkPos = cChunkDef::BlockToChunk(a_BlockPos);
+	auto relPos = cChunkDef::AbsoluteToRelative(a_BlockPos, chunkPos);
```

Comment:
The style guide: "am I a joke to you"?

Reply:
>  This will break all plugins, so as a side note, I have a proposition. The naming convention of Cuberite classes has been bothering me since forever, and this is the perfect moment to make things right - we're already breaking compatibility, so we can break it thoroughly. How about dropping the "c" prefix to all classes? And while we're at it, how about switching to a style that has long been used and is proven good? Personally, I like the Java-like style with membership prefixes (ClassName, doSomethingFunction(), mMemberVariable, aParameter)

Source: https://forum.cuberite.org/thread-3215.html

Reply:
Yeah but some aren't as keen with Java style :)

Reply:
It's definitely more sane than what we have now. And it's long proven. Qt uses it, too, in C++ world, mind you.

Reply:
I see no reason why the first word should be lowercase, as if its somehow less important. Who came up with this style anyway?